docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1380,7 +1380,8 @@ Affects the pruning of models that have not been updated for the given time
 duration. The value must be set to a multiple of the `bucket_span`. If set too
 low, important information may be removed from the model. Typically, set to
 `30d` or longer. If not set, model pruning only occurs if the model memory
-status reaches the soft limit or the hard limit.
+status reaches the soft limit or the hard limit. For jobs created in 8.1 and
+later, the default value is the greater of `30d` or 20 * `bucket_span`.
```

Comment:
@lcawl do you have a view on whether it should be "20 times `bucket_span`" instead of "20 * `bucket_span`"?

Reply:
I'd vote for the former for clarity.